Makefile

```diff
@@ -16,5 +16,6 @@ watch: yarn-install
 build: yarn-install
 	$(DOCKER_RUN) $(DOCKER_IMAGE_TAG) yarn gulp create-dist
 
-deploy: yarn-install
-	$(DOCKER_RUN) -e PORT -e HOSTNAME -v "$${SSH_AUTH_SOCK}":/ssh-auth.sock:ro -e SSH_AUTH_SOCK=/ssh-auth.sock $(DOCKER_IMAGE_TAG) yarn gulp deploy
+deploy: build
```

Comment:
let's use the `rsync` command instead of a very stable "0.0.8" version of a library JS.